skimage/transform/radon_transform.py

```diff
@@ -92,28 +88,19 @@ def radon(image, theta=None, circle=True):
         pad_width = [(pb, p - pb) for pb, p in zip(pad_before, pad)]
         padded_image = np.pad(image, pad_width, mode='constant',
                               constant_values=0)
+
     # padded_image is always square
     if padded_image.shape[0] != padded_image.shape[1]:
         raise ValueError('padded_image must be a square')
-    radon_image = np.zeros((padded_image.shape[0], len(theta)))
     center = padded_image.shape[0] // 2
+    radon_image = np.zeros((padded_image.shape[0], len(theta)))
 
-    shift0 = np.array([[1, 0, -center],
-                       [0, 1, -center],
-                       [0, 0, 1]])
-    shift1 = np.array([[1, 0, center],
-                       [0, 1, center],
-                       [0, 0, 1]])
-
-    def build_rotation(theta):
-        T = np.deg2rad(theta)
-        R = np.array([[np.cos(T), np.sin(T), 0],
-                      [-np.sin(T), np.cos(T), 0],
+    for i, T in enumerate(np.deg2rad(theta)):
+        cos_T, sin_T = np.cos(T), np.sin(T)
+        R = np.array([[cos_T, sin_T, -center * (cos_T + sin_T - 1)],
```

Comment:
Please, use lowercase for the variable names.

Reply:
Done. Thank you @soupault for your review.